middleware/admin-auth.js

```diff
@@ -0,0 +1,12 @@
+export default (req, res, next) => {
```

Comment:
also this is not a good way to create a middleware

Reply:
hmmm...i will work on it